### DISCUSSION TITLE
4/21(금) 복습

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#95] [General]
오늘은 그동안 배운 것들을 복습하는 날입니다.
오늘 이후부터의 과제는 제가 내도록 하겠습니다.
동아리가 다음주 금요일에 있는데 그때는 무조건 todo가 끝나 있어야합니다. 할 것이 많아요.
시험 기간이지만 너무 iOS를 재끼면 힘들어져요...😈

**Comment by Eunho0922** [comment]
넵!
